doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -102,3 +102,30 @@ To read more about the FFS technology and its compatibility with Matter, see the
 
 * `Matter Simple Setup for Wi-Fi Overview`_
 * `Matter Simple Setup for Thread Overview`_
+
+Reaction to the last Matter fabric removal
+==========================================
+
+When a Matter device is commissioned by the Matter controller it starts belonging to a specific Matter fabric.
```

Comment:
```suggestion
When a Matter device is commissioned by the Matter controller, it then belongs to a specific Matter fabric.
```
This section is worded slightly different from the similar section on the other page. Check that they don't contradict each other.

You could also consider using an include from one page to the other.

Reply:
I've tried to include that part. Did I do it correctly? :)